apps/cli/src/args.ts

```diff
@@ -0,0 +1,40 @@
+import { parseArgs } from "node:util";
+
+export const showHelp = () => console.log(HELP);
```

Comment:
I'd use console.info for output. log is often used for debug. At least in my experience.

Reply:
oh this is a nice catch, I didn't know about console.info, we should prevent console.log from linter, so debugging statements don't get commited by accident and only commit console.info where needed